chapters/7-publication.tex

```diff
@@ -229,7 +229,7 @@ \subsection{Getting started with {\LaTeX} as a team}
 
 %----------------------------------------------------
 
-\section{Preparing and publishing data}
+\section{Preparing research data}
```

Comment:
```suggestion
\section{Preparing research data for publication}
```
"Preparing research data" could be read as preparing for analysis, as in data processing.